catalog/task/create-github-release/0.1/tests/test-create-github-release.yaml

```diff
@@ -0,0 +1,68 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-create-github-release
+spec:
+  description: |
+    Run the create-github-release task
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/hacbs-release/release-utils:85ab98a7ec63c3d8d9ec3c4982ff0e581bcb3c83
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              mkdir $(workspaces.data.path)/release/
+              cat > $(workspaces.data.path)/release/foo.json << EOF
+              { "example github release file": "just mock data" }
+              EOF
+              touch $(workspaces.data.path)/release/foo.zip
+              touch $(workspaces.data.path)/release/foo_SHA256SUMS
+    - name: run-task
+      taskRef:
+        name: create-github-release
+      params:
+        - name: githubSecret
+          value: test-create-github-release-token
+        - name: repository
+          value: foo/bar
+        - name: release_version
+          value: 1.2.3
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      runAfter:
+        - setup
+    - name: check-result
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: check-result
+            image: quay.io/hacbs-release/release-utils:85ab98a7ec63c3d8d9ec3c4982ff0e581bcb3c83
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              if [ $(cat $(workspaces.data.path)/mock_gh.txt | \
+                  grep 'Mock gh' | wc -l) != 1 ]; then
+                echo Error: gh was expected to be called 1 time. Actual calls:
+                cat $(workspaces.data.path)/mock_gh.txt | grep 'Mock gh'
```

Comment:
The `grep` is not really needed - nothing else is saved to the file anyway (and I suggest removing the prefix in another comment).